### ISSUE TITLE
Add automated unit tests

### ISSUE THREAD

**Issue by SLaks** [SLaks/Styliner#1]
The ideal test for Styliner is to render inlined and non-inlined versions of complex pages in a browser, and make sure that the applied styles are identical.  (see tests/TestServer.js)
This requires a browser with a full CSS stack to test against; I'll use PhantomJS for that.

**Comment by stevenvachon** [comment]
Without this, no one can contribute. Really sad.

**Comment by SLaks** [comment]
Yes; the problem is that such tests aren't so simple to write, and I don't have time to write them.
If you want to write them yourself, I'd welcome that.

Also note that now that [Gmail supports CSS](https://developers.google.com/gmail/design/css), this project should no longer be necessary.

**Comment by stevenvachon** [comment]
Not all Gmail products support CSS yet, though: https://litmus.com/blog/gmail-to-support-responsive-email-design/

This project still has at least a few months left. Also, who knows what the future holds.
